test/common/proxy-server.js

```diff
@@ -74,8 +100,46 @@ exports.createProxyServer = function() {
   return { proxy, logs };
 };
 
-exports.checkProxiedRequest = async function(envExtension, expectation) {
-  const { spawnPromisified } = require('./');
+function spawnPromisified(...args) {
+  const { spawn } = require('child_process');
+  let stderr = '';
+  let stdout = '';
+
+  const child = spawn(...args);
+  child.stderr.setEncoding('utf8');
+  child.stderr.on('data', (data) => {
+    console.error('[STDERR]', data);
```

Comment:
These logs are added for debugging, otherwise the tests can be difficult to debug when they fail with no output from the child and I found myself constantly adding these logs.